bedrock/firefox/templates/firefox/home/index-master.html

```diff
@@ -134,7 +134,7 @@ <h4>{{ ftl('firefox-home-get-trackers-off') }}</h4>
       <div id="test-menu-browsers-wrapper" class="mzp-c-menu-list mzp-t-cta mzp-t-download">
         <h5 class="mzp-c-menu-list-title">{{ ftl('firefox-home-download-the-browser') }}</h5>
         <ul class="mzp-c-menu-list-list" id="menu-browsers">
-          {# Old IE users need to click a download button, the JS on the thank you page doesn't get them the right download if we send them there directly #}
+          {# Direct legacy IE users to the /new/ page where they can read EOL messaging see issue 13317 #}
```

Comment:
If the visitor is on IE 11 they won't see any link for desktop in the drop down menu because they fall in the space between support for the direct link and support for conditional comments (they only go up to IE10).

Reply:
I’m not sure I follow. IE11 is on Windows 10/11 only, so should not be effected by these changes. Unless I’m misunderstanding?

Reply:
At least, if IE 11 is available for Windows 8, then the expected behaviour would be to not see a desktop download link

Reply:
IE 11 is in the list for 8.1 on Browser Stack 🤷‍♀️

Reply:
It looks a little weird but it's not technically broken: 

<img width="611" alt="Screenshot 2023-07-18 at 11 20 39" src="https://github.com/mozilla/bedrock/assets/854701/1897b4fe-5d2a-494f-9fc3-82d7c60f2b50">

Reply:
Yeah, I don’t think I’m going to spend hours fitting messaging into non-standard places where we’ve chosen to not use download buttons.

Reply:
The conditional comment here sends IE6-10 users to the /new page to read the EOL messaging. If you remove the comments and display it contingent on .fx-unsupported we can give that experience to Edge & IE 11 too.

Alternatively, I suggest removing the conditional comment entirely so that the IE6-11 experience is the same.

Reply:
Hmm interesting suggestion! I'll take a look here and give it a try :)